README.md

```diff
@@ -0,0 +1,18 @@
+# map-printer
+
+Export Mapbox maps in the browser.
+
+Download massive hi-res PNGs from Mapbox map styles without memory limits.
+
+[https://map-printer.ddem.us](https://map-printer.ddem.us/)
+
+## Usage
+
+```
+git clone git@github.com:digidem/map-printer.git
```

Comment:
Actually this should be the http git link

Reply:
nice catch, done